src/async_jobs.ts

```diff
@@ -102,6 +141,21 @@ async function slackInteractivityHandler(
           `SERVER POST /slack-interactivity: Determined user interaction is a message shortcut.`
         );
 
+        if (payload.callback_id === 'set_needs_attention') {
```

Comment:
can you make this a switch statement on the callback_id (and I believe the below logic is callback_id = reset_demo)?